### PR TITLE
Tldraw - optimize placement on node import 

### PR DESCRIPTION
**Pull request by mdroidian**
https://www.loom.com/share/3c109f4a076c4143acaa2098b2554038

### CODE REVIEW

File: src/components/Export.tsx [hunk 1]
```diff
@@ -170,10 +176,65 @@ const ExportDialog: ExportDialogComponent = ({
...
+    };
+    const shapeBounds = extractShapesBounds(tldraw);
+
+    type CommonBounds = {
+      top: number;
+      right: number;
+      bottom: number;
+      left: number;
+    };
+    const findCommonBounds = (shapes: ShapeBounds[]): CommonBounds => {
```
Comment (dvargas92495) [inline src/components/Export.tsx:211]: surprised tldraw doesnt expose something for this

**Review by dvargas92495 — APPROVED** [review]

File: src/components/Export.tsx (hunk 1 above)
Comment (mdroidian) [reply src/components/Export.tsx:211]: They do, `app.allShapesCommonBounds` but I don't know how to get the `app` (the tlddraw instance of a canvas that isn't loaded) in `Export.tsx`.  

Did a little digging and found `useEditor`, but I believe you can only use that in children of the tldraw component.

Comment (dvargas92495) [reply src/components/Export.tsx:211]: oh damn. is this method just a copy paste of what they're doing in there?

Comment (mdroidian) [reply src/components/Export.tsx:211]: Nah, I figured it was easy enough to make so just did it.

**Pull request merged**